projects/demo/src/modules/components/slider-old/examples/import/declare-form.txt

```diff
@@ -0,0 +1,12 @@
+import {FormControl, FormGroup} from '@angular/forms';
```

Comment:
what do you think about using markdown? as it is in the example with the avatar demo

Reply:
it is `slider-old`, let leave legacy files as it is (they will be deleted in next major version).

But i will fix all new files, thanks

Reply:
thank you)